stdlib/experimental/aggregate/aggregate_test.flux

```diff
@@ -0,0 +1,116 @@
+,,10,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:00Z,0,bytes_recv,net,host.local,en2
+,,10,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:10Z,0,bytes_recv,net,host.local,en2
+,,10,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:20Z,0,bytes_recv,net,host.local,en2
+,,10,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:30Z,0,bytes_recv,net,host.local,en2
+,,10,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:40Z,0,bytes_recv,net,host.local,en2
+,,10,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:50Z,0,bytes_recv,net,host.local,en2
+,,11,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:00Z,17962,bytes_recv,net,host.local,utun2
+,,11,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:10Z,17962,bytes_recv,net,host.local,utun2
+,,11,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:20Z,17962,bytes_recv,net,host.local,utun2
+,,11,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:30Z,17962,bytes_recv,net,host.local,utun2
+,,11,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:40Z,17962,bytes_recv,net,host.local,utun2
+,,11,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:50Z,17962,bytes_recv,net,host.local,utun2
+,,12,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:00Z,11142341845,bytes_recv,net,host.local,en0
+,,12,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:10Z,11142346094,bytes_recv,net,host.local,en0
+,,12,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:20Z,11142349857,bytes_recv,net,host.local,en0
+,,12,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:30Z,11142355573,bytes_recv,net,host.local,en0
+,,12,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:40Z,11142361285,bytes_recv,net,host.local,en0
+,,12,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,2020-02-20T23:00:50Z,11142366838,bytes_recv,net,host.local,en0
+"
+
+outData = "
+#group,false,false,true,true,true,false,false
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,string,double,dateTime:RFC3339
+#default,out-data,,,,,,
+,result,table,_start,_stop,host,_value,_time
+,,0,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,host.local,,2020-02-20T23:00:10Z
+,,0,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,host.local,2875.8,2020-02-20T23:00:20Z
+,,0,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,host.local,5314.5,2020-02-20T23:00:30Z
+,,0,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,host.local,978.9000000000001,2020-02-20T23:00:40Z
+,,0,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,host.local,22495,2020-02-20T23:00:50Z
+,,0,2020-02-20T23:00:00Z,2020-02-20T23:01:00Z,host.local,7066.2,2020-02-20T23:01:00Z
+"
+
+t_rate = (table=<-) =>
+    table
+        |> range(start: 2020-02-20T23:00:00Z, stop: 2020-02-20T23:01:00Z)
+        |> filter(fn: (r) => r._measurement == "net" and r._field == "bytes_recv")
+        |> aggregate.rate(every: 10s, groupColumns: ["host"], unit: 1s)
```

Comment:
I'd like to see the test case use an `every` interval that is larger than the raw data interval so we can validate that the `mean` aggregation step is working as expected.

Reply:
Could you also include data for at least two hosts? So we can see that the computation preserves each host?